File: 628. Maximum Product of Three Numbers/solution.cs
public class Solution {
    public int MaximumProduct(int[] nums) {
        int product1, product2, res, n = nums.Length;
        Array.Sort(nums);

        product1 = nums[0]*nums[1]*nums[n-1];
        product2 = nums[n-1]*nums[n-2]*nums[n-3];
        
        res = Math.Max(product1,product2);

        return res;
    }
}

File: 628. Maximum Product of Three Numbers/solution.ts
function maximumProduct(nums: number[]): number {
  let n: number = nums.length,
    maxRes: number,
    product1: number,
    product2: number;

  nums.sort((a, b) => a - b);

  product1 = nums[0] * nums[1] * nums[n - 1];
  product2 = nums[n - 1] * nums[n - 2] * nums[n - 3];

  maxRes = Math.max(product1, product2);

  return maxRes;
}
